tool_for_bioinformatics.py

```diff
@@ -0,0 +1,153 @@
+
+        Returns:
+            If one sequence is supplied, a string with the result is returned.
+            If several are submitted, a list of strings is returned.
+            Depending on the function performed, the following returns will occur:
+                molecular_weight (int) or (list): amino acid sequence molecular weight number or list of numbers
+                three_letter_code (str) or (list): translated sequence from one-letter in three-letter code
+                length (int) or (list): integer number of amino acid residues
+                folding (str) or (list): 'alpha_helix', if there are more alpha helices
+                                        'beta_sheet', if there are more beta sheets
+                                        'equally', if the probability of alpha spirals and beta sheets are the same
+                seq_charge(str) or (list): "positive", "negative" or "neutral"
+            If seq contains symbols differ from IUPAC 1letter code for 22 proteinogenic amino acids,
+            the result for this sequence will be 'unexpected symbols in sequence' regardless of function
+            If action is not in the function the message 'unexpected action' will occur
+    """
+    *seqs, function = args
+    d_of_functions = {'molecular_weight': aa.molecular_weight,
+                      'three_letter_code': aa.three_letter_code,
+                      'length': aa.length,
+                      'folding': aa.folding,
+                      'seq_charge': aa.seq_charge}
+    answer = []
+    if function not in d_of_functions.keys():
+        raise ValueError("unexpected action")
+    for sequence in seqs:
+        if not set(sequence).issubset(AMINO_ACIDS):
+            answer.append('unexpected symbols in sequence')
+        else:
+            answer.append(d_of_functions[function](sequence))
+    if len(answer) == 1:
+        return answer[0]
+    else:
+        return answer
+
+
+def fastq_filtration(input_fastq, gc_bounds=(0, 100), length_bounds=(0, 2 ** 32), quality_treshold=0, output_fastq=''):
+    """
+    This function provides you the opportunity to filter the FASTQ file to select sequences
+    that fit  requirements on 5 parameters: input and output(optional) files, length, GC composition,
+    and quality of the reed
+    :parameters
+        input_fastq: path to fastq file
+        gc_bounds: (tuple) interval for the of acceptable GC content, in %
+        length_bounds: (tuple) interval for the of acceptable sequense length in number of nucleotide
+        quality_treshold: (float) threshold value for average quality per nucleotide (phred33 scale)
+        output_fastq = name of output file, ./fastq_filtrator_resuls/output_fastq, if it is not defined,
+        it will be the same of the input file
+
+    """
+    if not os.path.isdir('fastq_filtrator_resuls'):
+        os.mkdir('fastq_filtrator_resuls')
+    if output_fastq == '':

```

Comment:
```suggestion
    if not output_fastq:
```
Ну а вообще с учетом прошло комментария:
```suggestion
    if output_fastq is None:
```